src/components/create-guild/templates.ts

```diff
@@ -0,0 +1,236 @@
+        },
+      },
+      {
+        type: "ALCHEMY_CONTRACT_DEPLOY_RELATIVE",
+        chain: "POLYGON",
+        data: {
+          txCount: 1,
+          timestamps: { minAmount: 31536000000 },
+        },
+      },
+    ],
+  },
+
+  {
+    name: "Onchain Citizen",
+    description: "",
+    visibility: Visibility.PUBLIC,
+    logic: "OR",
+    imageUrl: `/guildLogos/${getRandomInt(286)}.svg`,
+    requirements: [
+      {
+        type: "ALCHEMY_TX_COUNT_RELATIVE",
+        chain: "ETHEREUM",
+        data: {
+          txCount: 1,
+          timestamps: { minAmount: 31536000000 },
+        },
+      },
+      {
+        type: "COVALENT_TX_COUNT_RELATIVE",
+        chain: "BASE_MAINNET",
+        data: {
+          txCount: 3,
+          timestamps: { minAmount: 31536000000 },
+        },
+      },
+      {
+        type: "COVALENT_TX_COUNT_RELATIVE",
+        chain: "OPTIMISM",
+        data: {
+          txCount: 3,
+          timestamps: { minAmount: 31536000000 },
+        },
+      },
+      {
+        type: "ALCHEMY_TX_COUNT_RELATIVE",
+        chain: "ARBITRUM",
+        data: {
+          txCount: 3,
+          timestamps: { minAmount: 31536000000 },
+        },
+      },
+      {
+        type: "ALCHEMY_TX_COUNT_RELATIVE",
+        chain: "POLYGON",
+        data: {
+          txCount: 3,
+          timestamps: { minAmount: 31536000000 },
+        },
+      },
+    ],
+  },
+  {
+    name: "Discord Fellow",
+    description: "",
+    visibility: Visibility.PUBLIC,
+    logic: "AND",
+    imageUrl: `/guildLogos/${getRandomInt(286)}.svg`,
+    requirements: [
+      {
+        type: "DISCORD_JOIN_FROM_NOW",
+        data: {
+          memberSince: 15724800000,
+        },
+      },
+    ],
+  },
+  {
+    name: "Sybil Resistant",
+    description: "",
+    visibility: Visibility.PUBLIC,
+    logic: "ANY_OF",
+    anyOfNum: 3,
+    imageUrl: `/guildLogos/${getRandomInt(286)}.svg`,
+    requirements: [
+      {
+        type: "EMAIL_VERIFIED",
+      },
+      {
+        data: {
+          schemaId:
+            "0xf8b05c79f090979bf4a80270aba232dff11a10d9ca55c4f88de95317970f0de9",
+          attester: "0x357458739F90461b99789350868CD7CF330Dd7EE",
+        },
+
+        type: "COINBASE_EAS_ATTESTED_BY",
+        chain: "BASE_MAINNET",
+      },
+      {
+        data: { minAmount: 100 },
+        type: "GUILD_ADMIN",
+        chain: "ETHEREUM",
+      },
+      {
+        type: "GITCOIN_PASS",
+        chain: "ETHEREUM",
+      },
+      {
+        data: { maxAmount: 2592000000, query: "" },
+        type: "POLYGON_ID_BASIC",
+        chain: "POLYGON",
+      },
+      {
+        type: "FARCASTER_PROFILE",
+
+        chain: "ETHEREUM",
+      },
+      {
+        data: { minAmount: 3 },
+
+        type: "SNAPSHOT_VOTES",
+        chain: "ETHEREUM",
+      },
+      {
+        type: "RABBITHOLE",
+        address: "0x2face815247a997eaa29881c16f75fd83f4df65b",
+        chain: "ETHEREUM",
```

Comment:
we need to set `data` as an empty object too, since there's a bug in gate right now and the guild won't get created unless we define that